homeassistant/components/knx/binary_sensor.py

```diff
@@ -63,24 +63,24 @@ def async_add_entities_discovery(hass, discovery_info, async_add_entities):
 @callback
 def async_add_entities_config(hass, config, async_add_entities):
     """Set up binary senor for KNX platform configured within platform."""
-    name = config.get(CONF_NAME)
+    name = config[CONF_NAME]
     import xknx
     binary_sensor = xknx.devices.BinarySensor(
         hass.data[DATA_KNX].xknx,
         name=name,
-        group_address=config.get(CONF_ADDRESS),
+        group_address_state=config[CONF_ADDRESS],
```

Comment:
It is a state address, so i think we should use `address_state` as the name of the value. I know that people have to update their configuration, but this would make the config more consistent.

Reply:
same for sensors (not binary_)?

Reply:
done for both.